### PR TITLE
testing for flakines in bl-update

### LINKED ISSUE #811: [flake] `bl-update` returns `missing blobber` error
Check the following screenshot - https://user-images.githubusercontent.com/9148277/216847256-1cbb37e9-8699-4b91-b340-c56385a1de32.png.

Latest occurence - https://github.com/0chain/system_test/actions/runs/4305246202/jobs/7507824703

This came up for the first time in #752

### PR DESCRIPTION
**Pull request by mayank-2105**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test: https://github.com/0chain/system_test/pull/644
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/handler.go [hunk 1]
```diff
@@ -2450,6 +2450,7 @@ func (srh *StorageRestHandler) getBlobber(w http.ResponseWriter, r *http.Request
 	}
 	blobber, err := edb.GetBlobber(blobberID)
 	if err != nil {
+		logging.Logger.Error("get blobber failed with error: ", zap.Error(err))
 		err := common.NewErrInternal("missing blobber: " + blobberID)
```
Comment (Kishan-Dhakan) [inline code/go/0chain.net/smartcontract/storagesc/handler.go]: Also return the actual error, no need to return this

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/smartcontract/storagesc/handler.go (hunk 1 above)
Comment (mayank-2105) [reply code/go/0chain.net/smartcontract/storagesc/handler.go]: Done in new commit. Please review and merge.

**Review by peterlimg — APPROVED** [review]
LGTM

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | bl_update_testcase  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | test_bl_update  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | bl_update_testcase  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | test_bl_update  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | bl_update_testcase  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | test_bl_update  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**